lldb/include/lldb/Host/aix/Ptrace.h

```diff
@@ -0,0 +1,44 @@
+//===-- Ptrace.h ------------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// This file defines ptrace functions & structures
+
+#ifndef liblldb_Host_aix_Ptrace_h_
+#define liblldb_Host_aix_Ptrace_h_
+
+#include <sys/ptrace.h>
+
+#define DEBUG_PTRACE_MAXBYTES 20
```

Comment:
What is the purpose of this and should it be declared like:
```
#ifndef DEBUG_PTRACE_MAXBYTES
#define ...
#endif
```
In case the build machine has a kernel that has changed this?

Reply:
This is a lldb-specific macro that shouldn't be here even on linux (it's just used in one place, and might as well be inlined there). Let's delete it from here.

Reply:
To be specific, it is being used in the `NativeProcessLinux.cpp`. For our context, it is used similarly for `NativeProcessAIX.cpp`, utilised by `DisplayBytes(..)<--PtraceDisplayBytes(..)`.
We can move it to the same file if you suggest so and in the same way as David suggested.

Reply:
For now, I can define it with the given value in that file locally and use it, and going forward we can update/get it removed if that is a better alternative.